### ISSUE TITLE
Creating a new fabric.Path from a command string inherits fabric.Object's default fill property

### ISSUE THREAD

**Issue by snemvalts** [kangax/fabric.js#2356]
Not setting the `fill` property of a new `fabric.Path` object to something empty causes the object to get filled with black color under the curve, the property is inherited from `fabric.Object`.

Here's a fiddle that demonstrates the issue: http://jsfiddle.net/dab9gm6j/3/
Draw a line -> Toggle hand -> Select line on canvas -> Copy selected line. The new, created line is filled with black color under the curve. Commenting in the `fill` property causes the line to look normal again.

I am using Fabric 1.5.

**Comment by asturur** [comment]
This is not an issue.
You are not copyng a path but creating a new one.
If you create a new object it is filled black. This is intended.

Exactly because what you see as a line, is a PATH filled with transparent.
Normally is expected to be filled with black.

If you clone the path  you get the same visual effect.
http://jsfiddle.net/dab9gm6j/4/

**Comment by snemvalts** [comment]
I never stated that I was copying the path, I was creating a new Path from the command string, which was required for me because I needed to load from a custom JSON file. `fill` property automatically being inherited (and being `rgb(0, 0, 0)`) is very much a gotcha, which I spent some time on to pinpoint and fix. This might be intended, but it can be very confusing to a new user of the library and should be changed.
